### PR TITLE
cleans up spawns in turrets

### PR DESCRIPTION
**Pull request by ghost**
SWEET SWEET GBP

### CODE REVIEW

**Comment by Bawhoppen** [comment]
>GDP

**Comment by lzimann** [comment]
this changelog is not needed at all

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -219,9 +223,7 @@
 			stat &= ~NOPOWER
 			update_icon()
 		else
-			spawn(rand(0, 15))
```
Comment (AnturK) [inline code/game/machinery/porta_turret/portable_turret.dm]: Why is this even here, just remove the delay.

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/game/machinery/porta_turret/portable_turret.dm (hunk 1 above)
Comment (Cyberboss) [reply code/game/machinery/porta_turret/portable_turret.dm]: Been wondering myself, a lot of power change events use random delays fnr

**Pull request closed without merging**